### ISSUE TITLE
My validate loss is unchanged when training, what's the reason may it be?

### ISSUE THREAD

**Issue by LaiPiXiong** [tensorflow/tensorflow#15679]
Dears,
        Lately, I train a Inception-v3 model in RAP_dataset. BUT, entropy loss  on Validate data is always unchanged when training. The loss shows as follows:

- sorry, I do not find the port attaching images. ........

And, yesterday, I add tf.control_dependencies(tf.get_collection(tf.GraphKeys.UPDATE_OPS)) in code, it still not work. What reasons may it be? Sincerely hope your replies, and BEST WISHES TO YOU!
